lib/python/ensembl/brc4/runnable/compare_fasta.py

```diff
@@ -364,74 +328,76 @@ def compare_seqs(self, seq1, seq2):
-                
-                # Only list sequences where the length is > 200
-                mini = {seq: name for seq, name in only1.items() if len(seq) <= 200}
-                maxi = {seq: name for seq, name in only1.items() if len(seq) > 200}
-                
-                if mini and len(mini) > 3000:
-                    comp.append("WARNING: Ignoring %d sequences from 1 with length <= 200" % len(mini))
-                    only1 = maxi
-                
-            if only1:
-                # Only list sequences where the length is > 1000
-                mini = {seq: name for seq, name in only1.items() if len(seq) <= 1000}
-                maxi = {seq: name for seq, name in only1.items() if len(seq) > 1000}
-
-                if mini and len(mini) > 3000:
-                    comp.append("WARNING: Ignoring %d sequences from 1 with length <= 1000" % len(mini))
-                    only1 = maxi
-            
-            if only1:
-                total = sum([len(seq) for seq in only1.keys()])
-                comp.append("WARNING: Sequences only in 1: %d (%d)" % (len(only1), total))
-                only_seq1 = {name: len(seq) for seq, name in only1.items()}
-                for name, length in sorted(only_seq1.items(), key=lambda x: x[1]):
-                    comp.append("\tOnly in 1: %s (%d)" % (name, length))
-                    
-            if only2:
-                stats["max_only2"] = max(len(seq) for seq in only2.keys())
-
-                # Only list sequences where the length is > 200
-                mini = {seq: name for seq, name in only2.items() if len(seq) <= 200}
-                maxi = {seq: name for seq, name in only2.items() if len(seq) > 200}
-                
-                if mini and len(mini) > 3000:
-                    comp.append("WARNING: Ignoring %d sequences from 2 with length <= 200" % len(mini))
-                    only2 = maxi
-                    
-            if only2:
-                # Only list sequences where the length is > 1000
-                mini = {seq: name for seq, name in only2.items() if len(seq) <= 1000}
-                maxi = {seq: name for seq, name in only2.items() if len(seq) > 1000}
-                
-                if mini and len(mini) > 3000:
-                    comp.append("WARNING: Ignoring %d sequences from 2 with length <= 1000" % len(mini))
-                    only2 = maxi
-
-            if only2:
-                total = sum([len(seq) for seq in only2.keys()])
-                comp.append("WARNING: Sequences only in 2: %d (%d)" % (len(only2), total))
-                only_seq2 = {name: len(seq) for seq, name in only2.items()}
-                for name, length in sorted(only_seq2.items(), key=lambda x: x[1]):
-                    comp.append("\tOnly in 2: %s (%d)" % (name, length))
-        
+        if only1:
+            stats["max_only1"] = max(len(seq) for seq in only1.keys())
```

Comment:
```suggestion
            stats["max_only1"] = max(only1, key=lambda k: len(k))
```
Using `max` function to its maximum potential (pun intended).